src/systems/log/LogPlayback.cc

```diff
@@ -209,6 +209,10 @@ void LogPlayback::Configure(const Entity &,
   // Prepend working directory if path is relative
   this->dataPtr->logPath = common::absPath(this->dataPtr->logPath);
 
+  // Set the entity offset.
+  // \todo This number should be included in the log file.
+  _ecm.SetEntityOffset(90000);
```

Comment:
How about ticketing an issue for keeping track of the max entity count during log record? 

Also, maybe a number like `std::numeric_limits<int64_t>::max() / 2` could be more generic than the current number.

Reply:
The issue is #347.

Added a test in 8a2b44cf1122c67287ac079a20efac4cb8ffd422